google/execute.ts

```diff
@@ -94,6 +97,28 @@ export function getStateChanges(command: ExecuteCommandTypes, params: any, devic
             }
             break;
 
+        case ExecuteCommandTypes.StartStop:
+        case ExecuteCommandTypes.PauseUnpause:
+            if (device.type === 'vacuum' && 'isRunning' in device.state) {
+                let running = device.state.isRunning;
+                let paused = device.state.isPaused;
+                const { start, pause } = params;
+                if (start != null) {
```

Comment:
Could start/pause come as undefined? Maybe a better check is `typeof start === 'boolean'`. Same for pause a few lines down.